.github/workflows/build.yml

```diff
@@ -22,46 +22,105 @@ env:
   RELEASE_OUT: "./release-out"
 
 jobs:
-  build:
+  test:
     runs-on: ubuntu-latest
     steps:
       -
         name: Checkout
         uses: actions/checkout@v3
-      -
-        name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
       -
         name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
         with:
           version: latest
       -
         name: Test
-        run: |
-          make test
+        uses: docker/bake-action@v2
+        with:
+          targets: test
+          set: |
+            *.cache-from=type=gha,scope=test
+            *.cache-to=type=gha,scope=test
       -
-        name: Send to Codecov
+        name: Upload coverage
         uses: codecov/codecov-action@v3
         with:
           file: ./coverage/coverage.txt
+
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.platforms.outputs.matrix }}
+    steps:
       -
-        name: Expose GitHub Runtime
-        uses: crazy-max/ghaction-github-runtime@906832f62b7baa936e3fbef72b029308af505ee7
```

Comment:
Can't quite work out where we using this before :eyes:, is this just a cleanup from existing code?

Reply:
Oh we were using it because the[ `make release` step](https://github.com/docker/buildx/blob/603595559f64233efa3052a1e825bdee500233e5/.github/workflows/build.yml#L53-L54) runs outside a GitHub Action (`run:`) and therefore requires to expose GitHub runtime to the workflow with https://github.com/crazy-max/ghaction-github-runtime